### PR TITLE
[v2.9] RKE2 August 2024 Patches 

### PR DESCRIPTION
**Pull request by rafaelbreno**

### CODE REVIEW

File: channels-rke2.yaml [hunk 1]
```diff
@@ -2211,3 +2298,35 @@ releases:
         repo: rancher-rke2-charts
         version: 3.3.0-rancher100
     featureVersions: *featureVersions-v1
+  - version: v1.30.4-rc2+rke2r1
```
Comment (ShylajaDevadiga) [inline channels-rke2.yaml]: needs update to rc3

Comment (rafaelbreno) [reply channels-rke2.yaml]: Updated!

File: channels-rke2.yaml [hunk 2]
```diff
@@ -1968,6 +1968,29 @@ releases:
         repo: rancher-rke2-charts
         version: 3.3.0-rancher100
     featureVersions: *featureVersions-v1
+  - version: v1.27.16+rke2r2
+    minChannelServerVersion: v2.7.11-alpha1
```
Comment (ShylajaDevadiga) [inline channels-rke2.yaml]: minChannelServerVersion: v2.8.0-alpha1?

File: channels-rke2.yaml [hunk 3]
```diff
@@ -1968,6 +1968,29 @@ releases:
...
+    maxChannelServerVersion: v2.8.99
```
Comment (ShylajaDevadiga) [inline channels-rke2.yaml]: v2.9.99

**Review by ShylajaDevadiga — APPROVED** [review]

**Pull request merged**